modules/getting-started/proc-techdocs-configure-odf-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-techdocs-configure-odf-operator_{context}"]
+
+= Making object storage accessible to containers using the Operator
```

Comment:
```suggestion
// Module included in the following assemblies:
//
// * assemblies/assembly-techdocs-plugin.adoc

:_mod-docs-content-type: PROCEDURE
[id="proc-techdocs-configure-odf-operator_{context}"]
= Making object storage accessible to containers using the Operator
```

For both of these headings (Operator and Helm), I'd update them to make it clearer if we mean containers that are using the Operator, or making storage accessible by using the Operator, by maybe updating it to:

- "Making object storage accessible to containers by using the Operator"
- "Making object storage accessible to containers by using the Helm chart"